src/controllers/pods.ts

```diff
@@ -149,8 +151,10 @@ export default function (server: Hapi.Server, deps: Injector) {
   async function getPodPrice (request: any, h: Hapi.ResponseToolkit) {
     const duration = request.query['duration'] || 3600
     log.debug('got pod options request. duration=' + duration)
-    const currencyPerSecond = getCurrencyPerSecond(deps)
-    const price = Math.ceil(currencyPerSecond * duration)
+    const currencyPerSecond = getCurrencyPerSecond(config)
+    log.debug(currencyPerSecond.toString())
```

Comment:
I think these logs were for our debugging, if we want to keep them we should add text and likely combine them together.